<?xml version="1.0" encoding="utf-8" standalone="no"?>
<test-results xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="nunit_schema_2.5.xsd" name="Pester" total="15" errors="0" failures="12" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2016-10-28" time="09:34:12">
  <environment user="crip" machine-name="DESKTOP-KHGP00I" cwd="C:\users\crip\Google Drive\Code\PowervRA\tests" user-domain="DESKTOP-KHGP00I" platform="Microsoft Windows 10 Home|C:\WINDOWS|\Device\Harddisk0\Partition3" nunit-version="2.5.8.0" os-version="10.0.14393" clr-version="4.0.30319.42000" />
  <culture-info current-culture="en-GB" current-uiculture="en-US" />
  <test-suite type="Powershell" name="Pester" executed="True" result="Failure" success="False" time="15.3444" asserts="0">
    <results>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test002-Tenant.Tests.ps1" executed="True" result="Failure" success="False" time="3.2333" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test002-Tenant.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test002-Tenant.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test002-Tenant.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test002-Tenant.Tests.ps1'" time="3.2333" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test003-ReservationPolicy.Tests.ps1" executed="True" result="Failure" success="False" time="0.8934" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test003-ReservationPolicy.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test003-ReservationPolicy.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test003-ReservationPolicy.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test003-ReservationPolicy.Tests.ps1'" time="0.8934" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test004-BusinessGroup.Tests.ps1" executed="True" result="Failure" success="False" time="0.4987" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test004-BusinessGroup.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test004-BusinessGroup.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test004-BusinessGroup.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test004-BusinessGroup.Tests.ps1'" time="0.4987" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test005-ServiceAndEntitlement.Tests.ps1" executed="True" result="Failure" success="False" time="0.6256" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test005-ServiceAndEntitlement.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test005-ServiceAndEntitlement.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test005-ServiceAndEntitlement.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test005-ServiceAndEntitlement.Tests.ps1'" time="0.6256" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test006-BlueprintAndServiceBlueprint.Tests.ps1" executed="True" result="Failure" success="False" time="0.3997" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test006-BlueprintAndServiceBlueprint.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test006-BlueprintAndServiceBlueprint.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test006-BlueprintAndServiceBlueprint.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test006-BlueprintAndServiceBlueprint.Tests.ps1'" time="0.3997" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test007-ContentPackage.Tests.ps1" executed="True" result="Failure" success="False" time="0.4527" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test007-ContentPackage.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test007-ContentPackage.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test007-ContentPackage.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test007-ContentPackage.Tests.ps1'" time="0.4527" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test008-Reservation.Tests.ps1" executed="True" result="Failure" success="False" time="1.4365" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test008-Reservation.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test008-Reservation.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test008-Reservation.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test008-Reservation.Tests.ps1'" time="1.4365" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test009-CatalogItem.Tests.ps1" executed="True" result="Failure" success="False" time="0.4782" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test009-CatalogItem.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test009-CatalogItem.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test009-CatalogItem.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test009-CatalogItem.Tests.ps1'" time="0.4782" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test010-Principal.Tests.ps1" executed="True" result="Failure" success="False" time="0.613" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test010-Principal.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test010-Principal.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test010-Principal.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test010-Principal.Tests.ps1'" time="0.613" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test099-Misc.Tests.ps1" executed="True" result="Failure" success="False" time="0.3695" asserts="0" description="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test099-Misc.Tests.ps1">
        <results>
          <test-case description="Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test099-Misc.Tests.ps1'" name="Error in C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test099-Misc.Tests.ps1.Error occurred in test script 'C:\users\crip\Google Drive\Code\PowervRA\tests\_archive\Test099-Misc.Tests.ps1'" time="0.3695" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>The remote server returned an error: (400) Bad Request.</message>
              <stack-trace>at Connect-vRAServer, C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1: line 120</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Module Tests" executed="True" result="Success" success="True" time="1.0135" asserts="0" description="Module Tests">
        <results>
          <test-case description="Attempting to import the PowervRA Module" name="Module Tests.Attempting to import the PowervRA Module" time="0.8769" asserts="0" success="True" result="Success" executed="True" />
          <test-case description="Checking xRequires Private Function exists" name="Module Tests.Checking xRequires Private Function exists" time="0.1366" asserts="0" success="True" result="Success" executed="True" />
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Connectivity Tests" executed="True" result="Failure" success="False" time="5.237" asserts="0" description="Connectivity Tests">
        <results>
          <test-case description="Attempting to ping the vRA Appliance vraap02.vrademo.local" name="Connectivity Tests.Attempting to ping the vRA Appliance vraap02.vrademo.local" time="4.7592" asserts="0" success="True" result="Success" executed="True" />
          <test-case description="Connects to a vRA Appliance and generates a token" name="Connectivity Tests.Connects to a vRA Appliance and generates a token" time="0.4778" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>{"errors":[{"code":90135,"message":"Unable to authenticate user TenantAdmin01@vrademo.local in tenant Tenant01.","systemMessage":"90135-Unable to authenticate user TenantAdmin01@vrademo.local in tenant Tenant01.","moreInfoUrl":null}]}</message>
              <stack-trace>at line: 120 in C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Connect-vRAServer.psm1</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
      <test-suite type="TestFixture" name="Disconnectivity Tests" executed="True" result="Failure" success="False" time="0.0935" asserts="0" description="Disconnectivity Tests">
        <results>
          <test-case description="Disconnects from a vRA Appliance" name="Disconnectivity Tests.Disconnects from a vRA Appliance" time="0.0935" asserts="0" success="False" result="Failure" executed="True">
            <failure>
              <message>vRA Connection variable does not exist. Please run Connect-vRAServer first to create it</message>
              <stack-trace>at line: 22 in C:\Users\crip\Documents\WindowsPowerShell\Modules\PowervRA\1.3.2\Functions\Public\Disconnect-vRAServer.psm1</stack-trace>
            </failure>
          </test-case>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>